lib/load.php

```diff
@@ -100,7 +100,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.7/post-formats.php';
 
 // WordPress 6.8 compat.
-require __DIR__ . '/compat/wordpress-6.8/admin-bar.php';
+// Note: admin-bar.php (69271) was reverted in Gutenberg 20.8.0. See https://github.com/WordPress/gutenberg/pull/69974.
```

Comment:
Why is this comment necessary?

Reply:
I added it because I knew this would not be the last change to this menu feature.
When I had to research the past changes, it was a bit of a pain to find all the history.